laalaa/apps/advisers/middleware.py

```diff
@@ -1,7 +1,8 @@
 from django.http import HttpResponse
+from django.utils.deprecation import MiddlewareMixin
 
 
-class PingMiddleware:
+class PingMiddleware(MiddlewareMixin):
```

Comment:
Could you briefly indicate in the PR summary what this mixin is for, please? And maybe what the deprecation timeline is for it, if indeed it does what I think it might.

Reply:
Thanks to your comment I have realised that django.utils.deprecation.MiddlewareMixin was for making it compatible with pre django 1.10 middleware spec.

I have now updated PingMiddleware to work with latest django middleware spec.